app/helpers/date_time_helper.rb

```diff
@@ -44,4 +45,14 @@ def api_date_format(date_time)
       end
     end
   end
+
+  def to_be_debated_on_in_words(date, today = Date.current)
+    return unless date.present?
```

Comment:
Yes I agree.. sorry, I forgot to remove it.. but it's all done now.. commit [4700c6f](https://github.com/alphagov/e-petitions/pull/456/commits/4700c6f01e13f3292772e6934e70973e2dac54d6) 👍 😄 tks